service/controller/rbac/rbac.go

```diff
@@ -44,14 +43,15 @@ func NewRBAC(config RBACConfig) (*RBAC, error) {
 
 	var namespaceAuthController *controller.Controller
 	{
-		selector := newSelector(func(labels labels.Labels) bool {
-			return labels.Has(label.Organization) || labels.Has(legacylabel.LegacyCustomer)
-		})
+		selector, err := labels.Parse(fmt.Sprintf("%s,%s", label.Organization, legacylabel.LegacyCustomer))
```

Comment:
The previous selector was OR `||`, but `,` means AND so this changes the logic. And according to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/, `Caution: For both equality-based and set-based conditions there is no logical OR (||) operator. Ensure your filter statements are structured accordingly.` We'll need to figure out the implication here.

Reply:
Taking a deeper look it seems here we are filtering `Namespaces` that are labeled with the `organization` label or with the `customer` label. On `kvm` installations I found namespaces using the `customer` label which doesn't use `organization` label. So I believe we have two options:
- We remove the `customer` part of the selector and we don't upgrade `rbac-operator` on kvm installations
- We remove the whole selector and add the filter in all resources executed by this controller

What do you think?

Reply:
I think it would be better to remove the selector and add a check to all resources.

Reply:
Moved selector to individual handlers, please take a look.

Reply:
If it's okay, can the check be in a function in the rbac package with a comment explaining what it does?

Reply:
Sure! Done.

Reply:
Thanks! :D Works for me.